### PR TITLE
Added modin.🐼 suppport.

### PR DESCRIPTION
**Pull request by rentruewang**
This is a squashed version of #129. Closes #129.

### CODE REVIEW

File: pygwalker/utils/gwalker_props.py [hunk 1]
```diff
@@ -271,6 +277,13 @@ def get_prop_getter(df: DataFrame) -> DataFramePropGetter:
         if isinstance(df, pl.DataFrame):
             __classname2method[pl.DataFrame] = _build_polars_prop_getter(pl)
             return __classname2method[pl.DataFrame]
+
+    if 'modin.pandas' in sys.modules:
+        from modin import pandas as mpd
+        if isinstance(df, mpd.DataFrame):
+            __classname2method[pd.DataFrame] = _build_pandas_prop_getter(pd)
```
Comment (longxiaofei) [inline pygwalker/utils/gwalker_props.py:284]: `__classname2method[mpd.DataFrame] = _build_pandas_prop_getter(mpd)`

File: pygwalker/utils/gwalker_props.py [hunk 2]
```diff
@@ -271,6 +277,13 @@ def get_prop_getter(df: DataFrame) -> DataFramePropGetter:
...
+            return __classname2method[pd.DataFrame]
```
Comment (longxiaofei) [inline pygwalker/utils/gwalker_props.py:285]: `return __classname2method[mpd.DataFrame]`

Comment (rentruewang) [reply pygwalker/utils/gwalker_props.py:285]: my bad

File: pygwalker/utils/gwalker_props.py [hunk 3]
```diff
@@ -142,13 +147,14 @@ def decode_fname(cls, s: Series, **kwargs) -> str:
 class PandasDataFramePropGetter(DataFramePropGetter[DataFrame]):
     pass
 
-
 class PolarsDataFramePropGetter(DataFramePropGetter[DataFrame]):
     pass
 
+class ModinDataFramePropGetter(DataFramePropGetter[DataFrame]):
```
Comment (longxiaofei) [inline pygwalker/utils/gwalker_props.py:153]: remove this class

**Comment by longxiaofei** [comment]
After the project refactoring, perhaps it will be easier for developers to participate in the development of pygwalker.😄

**Comment by longxiaofei** [comment]
After I finish test code, merge this PR into the main branch.

**Review by longxiaofei — APPROVED** [review]

**Pull request merged**